ai-server/app/services/project_recommendation_service.py

```
@@ -0,0 +1,455 @@
+import random
+from typing import List, Dict, Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)